README.md

```diff
@@ -38,22 +46,24 @@ spec:
         image: busybox:latest
         name: busybox
 ```
-Resultant deployment will have secrets present under path /etc/secret-sneaker-data/
+Resultant deployment will have secrets present under path /etc/secret-sneaker-data/ inside the pod container
```

Comment:
can this path be taken from user?

Reply:
also what do I have to do, if I have to just mount one key value pair from the secret?

Reply:
> can this path be taken from user?

Hey @viveksinghggits I am not sure how we can get the path dynamically from the user without a templating tool like Ansible or Terraform. I tried to use the Labels/Annotations key:values for the path as well but Valid value regex doesn't allow "/" character in the string.